### DISCUSSION TITLE
Can I run this on an anycubic Kobra 2 Neo?

### DISCUSSION THREAD

**Discussion by AfonsoTenda** [1coderookie/Klipper4KobraGoNeo#13] [Q&A]
I was just wondering if I Can  run this on an anycubic Kobra 2 Neo

**Comment by 1coderookie** [comment]
**No!**  
I set up another repo for the K2 series (https://github.com/1coderookie/Klipper4Kobra2series), but there are no files available yet. Until sth will be available there, you can find the necessary files for the K2Neo here for example: https://www.reddit.com/r/anycubic/comments/1987n78/kobra_2_neo_running_klipper_with_config_and_guide/  

Also check out my infosite about the K2Neo, might come in handy one day: https://1coderookie.github.io/Kobra2NeoInsights/  ;)
